validator/src/main.rs

```diff
@@ -2057,6 +2057,18 @@ pub fn main() {
                         .default_value(DEFAULT_RELAYER_MAX_FAILED_HEARTBEATS)
                 )
         )
+        .subcommand(
+            SubCommand::with_name("set-shred-receiver-address")
+                .about("Changes shred receiver address")
+                .arg(
+                    Arg::with_name("shred_receiver_address")
+                        .long("shred-receiver-address")
+                        .value_name("SHRED_RECEIVER_ADDRESS")
+                        .takes_value(true)
+                        .help("Shred receiver listening address. Set to empty string to disable.")
```

Comment:
this is kinda misleading/shred receiver address kinda misleading in general. wish we would've named it shred forwarder address... can you make this help string more clear its an address we forward shreds to?